src/android/OSAppFeedback.java

```diff
@@ -98,6 +98,7 @@ public void run() {
                 RelativeLayout.LayoutParams params = new RelativeLayout.LayoutParams(RelativeLayout.LayoutParams.MATCH_PARENT, RelativeLayout.LayoutParams.MATCH_PARENT);
                 ectViewGroup.setLayoutParams(params);
                 ectViewGroup.setId(2001);
+                ectViewGroup.setFitsSystemWindows(true);

```

Comment:
Curious this should work. I found setting this programmatically extremely finicky in other situations but then again what do I know... Good job 👍